reducemathview/src/main/assets/katex/contrib/copy-tex.js

```diff
@@ -1,208 +1,258 @@
+          enumerable: true,
+          get: getter,
+        });
+        /******/
       }
-
-      texSource.innerHTML = copyDelimiters.inline[0] + texSource.innerHTML + copyDelimiters.inline[1];
-    }
-  } // Switch display math to display delimiters.
-
-
-  var displays = fragment.querySelectorAll('.katex-display annotation');
-
-  for (var _i2 = 0; _i2 < displays.length; _i2++) {
-    var _element2 = displays[_i2];
-    _element2.innerHTML = copyDelimiters.display[0] + _element2.innerHTML.substr(copyDelimiters.inline[0].length, _element2.innerHTML.length - copyDelimiters.inline[0].length - copyDelimiters.inline[1].length) + copyDelimiters.display[1];
-  }
-
-  return fragment;
-};
-/* harmony default export */ var katex2tex = (katexReplaceWithTex);
-// CONCATENATED MODULE: ./contrib/copy-tex/copy-tex.js
-
- // Global copy handler to modify behavior on .katex elements.
-
-document.addEventListener('copy', function (event) {
-  var selection = window.getSelection();
-
-  if (selection.isCollapsed) {
-    return; // default action OK if selection is empty
-  }
-
-  var fragment = selection.getRangeAt(0).cloneContents();
-
-  if (!fragment.querySelector('.katex-mathml')) {
-    return; // default action OK if no .katex-mathml elements
-  } // Preserve usual HTML copy/paste behavior.
-
-
-  var html = [];
-
-  for (var i = 0; i < fragment.childNodes.length; i++) {
-    html.push(fragment.childNodes[i].outerHTML);
-  }
-
-  event.clipboardData.setData('text/html', html.join('')); // Rewrite plain-text version.
-
-  event.clipboardData.setData('text/plain', katex2tex(fragment).textContent); // Prevent normal copy handling.
-
-  event.preventDefault();
+      /******/
+    }; // define __esModule on exports
+    /******/
+    /******/ /******/ __webpack_require__.r = function (exports) {
+      /******/ if (typeof Symbol !== "undefined" && Symbol.toStringTag) {
+        /******/ Object.defineProperty(exports, Symbol.toStringTag, {
+          value: "Module",
+        });
+        /******/
+      }
+      /******/ Object.defineProperty(exports, "__esModule", { value: true });
+      /******/
+    }; // create a fake namespace object // mode & 1: value is a module id, require it // mode & 2: merge all properties of value into the ns // mode & 4: return value when already ns object // mode & 8|1: behave like require
+    /******/
+    /******/ /******/ /******/ /******/ /******/ /******/ __webpack_require__.t = function (
+      value,
+      mode
+    ) {
+      /******/ if (mode & 1) value = __webpack_require__(value);
+      /******/ if (mode & 8) return value;
+      /******/ if (
+        mode & 4 &&
+        typeof value === "object" &&
+        value &&
+        value.__esModule
+      ) {
+        return value;
+      }
+      /******/ const ns = Object.create(null);
+      /******/ __webpack_require__.r(ns);
+      /******/ Object.defineProperty(ns, "default", {
+        enumerable: true,
+        value: value,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/reduce-algebra/reduce-native/pullRequest?prid=7267754)
```suggestion
        value,
```